### ISSUE TITLE
The decision of showing sidebar permanently on tablet?

### ISSUE THREAD

**Issue by kuanyui** [TeamAmaze/AmazeFileManager#1699]
I have two Android tablets (Both installed Amaze file manager 3.3.4, via Google Play):

1. ASUS ZenPad 3s 10 (P027), resolution = (1536, 2048)
2. Sansumg Galaxy Tab S6 (SM-T860), resolution = (1600, 2560)

The previous one (ZenPad) has smaller resolution, but can always show the sidebar permantely; while the later one has larger resolution, but can't show like that, very inconvenient for a tablet:

![P_20191008_181543](https://user-images.githubusercontent.com/1370070/66389581-4e3a0880-e9fb-11e9-95d8-585689becd2f.jpg)

**Comment by EmmanuelMess** [comment]
@VishalNehra i invoke you

**Comment by VishalNehra** [comment]
Moved to 3.6 as we don't have bandwidth for this ATM

**Comment by kuanyui** [comment]
![Screenshot_20201008-015649_Settings](https://user-images.githubusercontent.com/1370070/95369164-aacf4c80-0909-11eb-9a71-961c2f78e20b.jpg)

![Screenshot_20201008-014455_Amaze](https://user-images.githubusercontent.com/1370070/95368123-3ea01900-0908-11eb-8eda-3abe201aaac7.jpg)
Sansumg Galaxy Tab S6 (SM-T860), issue seems disappeared at AmazeFileManager v3.4.3 , as long as adjusting system display zooming to minimum value?

**Comment by VishalNehra** [comment]
Yes, it's basically dpi dependent. Display zooming changes the dpi.. let me see if I can define one for your device default value, shouldn't be difficult.

**Comment by shuvashish76** [comment]
Device details: 
model = Lenovo Tab 4 8 Plus (TB-8704X) android version = 8.1.0 (stock os)
resolution = (1200, 1920) ratio =0.625
system Font size set to Smail (minimum value)
---
Portrait mode doesn't show sidebar permanently but in Landscape mode it does.

![amaze_sidebar](https://user-images.githubusercontent.com/48226312/95408132-7032d780-093c-11eb-83b3-a323316e8112.gif)

**Comment by VishalNehra** [comment]
Tried but couldn't find the solution. @kuanyui  Please use lowest display size for now.
@shuvashish76 I tried on 7" opened navigation drawer didn't really look good. Not much space was left usable.
